src/bde_copy.cpp

```diff
@@ -1519,8 +1527,16 @@ bool read_args( char *image, int argc, char *argv[] )
                       help(image); break;
 
             default:
-                printf("Invalid option %s\n",arg);
-                argsok = false;
+            if ( infiles && ! arg[1] && ! outfile && ! metafile) 
+            {
+                  output = true;
+                  meta = stderr;
+            }
+            else
+            {
+                  printf("Invalid option %s\n",arg);
+                  argsok = false;
+            }
```

Comment:
I think it would be cleaner to just check "outputfile" to have the value "-" at the end of commandline parsing.
The above condition is hard to read, what do you mean by `! arg[1]` ? What is that supposed to do ?

Reply:
I have moved the condition that I previously had to above the switch statement to try make it clearer and added a few comments to explain. I used `! arg[1]` to determine that the argument entered was only "-" rather than being an option or invalid option. It would be cleaner to check the outputfile value at the end but I don't see how that could be done, when if the option "-" is added it enters the switch block first because of the options starting with "-".

Reply:
It's maybe too much for this PR but in the future it could be good to use the standard `getopt` function from unistd.h (man 3 getopt)